### PR TITLE
Add Bot API Version Status Badge

### PR DESCRIPTION
**Pull request by kamikazechaser**
**Ref:** #260 
**Info:** Displays the Bot API version as a status badge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
 [![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-telegram-bot-api)
```
Comment (GochoMugo) [inline README.md]: indent this by one space, please!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
```
Comment (GochoMugo) [inline README.md]: I think this should point to https://core.telegram.org/bots/api

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
-[![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-telegram-bot-api)
+ [![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://core.telegram.org/bots/api)
```
Comment (GochoMugo) [inline README.md]: sorry, the 1st line is **not** indented!

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+ [![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-  telegram-bot-api)
```
Comment (GochoMugo) [inline README.md]: is there some space after `node-` in the link?

**Comment by kamikazechaser** [comment]
Fixed [indentation](https://raw.githubusercontent.com/kamikazechaser/node-telegram-bot-api/67c16adcd0f10b772c3a17355f0d3b253b5a7c84/README.md) and changed [re-direct url](https://core.telegram.org/bots/api)

**Pull request merged**